### DISCUSSION TITLE
Downloading extensions from the Success Community through Mirth

### DISCUSSION THREAD

**Discussion by antoinetielbeke** [nextgenhealthcare/connect#5138] [Feature/Enhancement Ideas]
We are setting up Mirth Connect in Kubernetes and we have several extensions. We want to use one of the Mirth properties `EXTENSIONS_DOWNLOAD` to download the extensions. But the official plugins like Advanced alerting or SSL Manager are in the NextGen Success Community which needs credentials to login. Is there a way I Mirth can authenticate itself? Or do I need to host these files myself? It would be great if I can just pull them from the Success Community.

Bonus question: I read the following in the documentation:
> /extensions: Libraries and meta data for Plug-ins and Connectors

Does this mean only the files for the extensions are stored here? If so, where is the configuration for the extensions stored? In the database?

**Comment by pacmano1** [comment]
I think someone asked that over on Slack, you need to host them yourselves at the moment.   Might open a tik with NG support also.

**Reply by antoinetielbeke** [reply]
That's a bummer. Do you know where the extensions configuration is stored? In the database or in the /extensions folder?

**Reply by jonbartels** [reply]
Technically both. The extension libraries themselves are in the `extensions` directory then each extension may use the DB and store its own configuration properties in the MC DB.

Extensions also physically can create their own DB tables though I'm only aware of one that does that in practice.
